### ISSUE TITLE
Dosbox-X XPx86 SDL2 slightly distorted MT-32 sound

### ISSUE THREAD

**Issue by pablokks** [joncampbell123/dosbox-x#4925]
### Describe the bug

I have experienced slightly distortion in sound while playing Red Baron with MT-32 set up as soundcard with Dosbox-X XPx86 SDL2. The music is mostly playing fine, but there is a small distorted sound like a little bit of sand would come through the speaker. Interestingly when I tested the other versions so x64/86 SDL1 and x64 SDL2 the music is flawless.

### Steps to reproduce the behaviour

1. Run SDL2 x86
2. Set mididevice=mt32 and create the path to roms
3. Run a game with MT-32 set up e.g. Red Baron

### What operating system(s) this bug have occurred on?

Windows 10

### What version(s) of DOSBox-X have this bug?

XPx86 SDL2 v2024.03.01

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by maron2000** [comment]
Yes it is a known bug (#3292), but its solution is yet to be found.
Updating SDL2 and mt32 libraries didn't work.

**Comment by Starfoxfs** [comment]
I have Dosbox-X 2025.02.01 on Windows VisualStudio x64 SDL2 Version and MT32 is fine.

Now i changed to Linux Mint 22.1 and use Dosbox-X Binaries from Github Action Page (Ubuntu Builds).

With SDL2 i have minimal distortion or crackling. SDL1 is more worst and even more distortion.

**Comment by Starfoxfs** [comment]
Its Interesting, if i move the Mouse fast from left to right there is more Distortion. I tested this with Lands of Lore 1
